docs/DeveloperGuide.md

```diff
@@ -274,7 +274,13 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* administrator of a small Non-Governmental Organisation (NGO) 
+* responsible for managing a large network of donors, volunteers, and partners
+* combined number of approximately 200 donors, volunteers and partners
+* has a need to manage volunteers participation hours 
+* has a need to regularly follow up with donors on monthly donations
```

Comment:
Perhaps this is not so appropriate since we don't really have a feature that complements this need?